corporate/views/portico.py

```diff
@@ -124,6 +124,9 @@ def communities_view(request: HttpRequest) -> HttpResponse:
         want_advertise_in_communities_directory=True
     ).order_by("name")
     for realm in want_to_be_advertised_realms:
+        # Filter out realms who haven't changed their description from the default.
+        if not realm.description:
+            continue
```

Comment:
Let's do this in the `filter` a couple lines above this, rather than in Python.  The `allow_web_public_streams_access` logic is complicated enough that it's easier to read in Python, but this limit is simple and there's no reason to do it in Python.

Reply:
Updated.